website/docs/reference/workspaces.md

```diff
@@ -100,6 +100,7 @@ Keys that are supported:
 - `readme`
 - `repository`
 - `cairo-version`
+- `experimental`
```

Comment:
I would rather **not** make it easy for users to enable experimental features for entire workspace. We should expect users to avoid using them in production code 99% of cases. Especially, I'd prohibit publishing packages with experimental features turned on.